karate-core/src/test/java/com/intuit/karate/JsonUtilsTest.java

```diff
@@ -39,7 +39,7 @@ void testDetect() {
     void testBeanConversion() {
         SimplePojo pojo = new SimplePojo();
         String s = JsonUtils.toJson(pojo);
-        assertEquals("{\"bar\":0,\"foo\":null}", s);
+        Match.that(s).isEqualTo("{\"bar\":0,\"foo\":null}");
```

Comment:
You can consider replacing the variable s with JsonUtils.toJson(pojo) for two reasons.
1. The name of variable s doesn't have any specific meaning
2. The variable s is only called once

Reply:
line 41 is the original code from the author; I didn't modify it. Also in the previous version the author also uses `s` to compare with ground truth.